packages/cli/src/commands/setup/shadowenv/shadowenv.js

```diff
@@ -0,0 +1,77 @@
+import fs from 'fs'
+import path from 'path'
+
+import boxen from 'boxen'
+import Listr from 'listr'
+
+import { getPaths, writeFile } from '../../../lib'
+import c from '../../../lib/colors'
+
+const shadowenvConfigPath = `${getPaths().base}/.shadowenv.d/rw.lisp`
+
+export const builder = (yargs) => {
+  yargs.option('force', {
+    alias: 'f',
+    default: false,
+    description: 'Overwrite existing configuration',
+    type: 'boolean',
+  })
+}
+
+export const handler = async ({ force }) => {
+  const tasks = new Listr(
+    [
+      {
+        title: 'Configuring Shadowenv...',
+        task: (_, task) => {
+          /**
+           * Check if Shadowenv config already exists.
+           * If it exists, throw an error.
+           */
+          if (!force && fs.existsSync(shadowenvConfigPath)) {
+            throw new Error(
+              'Shadowenv config already exists.\nUse --force to override existing config.'
+            )
+          } else {
+            return writeFile(
+              shadowenvConfigPath,
+              fs
+                .readFileSync(
+                  path.resolve(__dirname, 'templates', 'rw.lisp.template')
+                )
+                .toString(),
+              {
+                overwriteExisting: force,
+              },
+              task
+            )
+          }
+        },
+      },
+      {
+        title: '',
+        task: (_, task) => {
+          task.title = `One more thing...\n\n ${boxen(
+            [
+              c.green('Installation steps for Shadowenv'),
```

Comment:
```suggestion
          task.title = `One more manual step...\n\n ${boxen(
            [
              c.green('Required Installation Steps for Shadowenv'),
```

Attempting to make language be more direct about a manual next step. Thoughts?